CHANGELOG.md

```diff
@@ -46,6 +46,14 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Add configuration files binding for `--nv` option, these files are added in
   the `nvliblist.conf` that are recognized in the NVIDIA Container Toolkit,
   including files for EGL ICD.
+- `apptainer.conf` now accepts setting the `allow uts ns` option, and can
+  invalidate the use of the `--uts` and `--hostname` flags.
+- `apptainer.conf` now accepts setting new options regarding namespaces:
```

Comment:
It should be made clear that each of these default to `yes` but when set to `no` they will do the following.